ui/src/clockface/components/dropdowns/MultiSelectDropdown.tsx

```diff
@@ -252,9 +252,12 @@ class MultiSelectDropdown extends Component<Props, State> {
   }
 
   private validateChildCount = (): void => {
-    const {children} = this.props
+    const {children, status} = this.props
 
-    if (React.Children.count(children) === 0) {
+    const shouldHaveChildren =
+      status === ComponentStatus.Default || status === ComponentStatus.Valid
+
+    if (shouldHaveChildren && React.Children.count(children) === 0) {
```

Comment:
@alexpaxton @mavarius The interface for the `MultiSelectDropdown` drifted a bit from the `Dropdown`. This was one change made only in the `Dropdown`.

Another, that I didn't update here, was that the `MultiSelectDropdown` expects the `value` prop of a `Dropdown.Item` to be an object with an `id` field. It will call the `onChange` handler with the `id`. This is different than the `Dropdown` component, which will call the `onChange` handler with whatever is passed to the `value` prop of each `Dropdown.Item`.